packages/schematics/angular/migrations/update-8/ts-script-target.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { JsonParseMode, parseJsonAst } from '@angular-devkit/core';
+import { Rule, Tree } from '@angular-devkit/schematics';
+import {
+  findPropertyInAstObject,
+  insertPropertyInAstObjectInOrder,
+} from '../../utility/json-utils';
+
+export function updateTsConfigScriptTarget(): Rule {
```

Comment:
We do not have a guarantee that this would enable differential loading.

We'd have to parse browserlist to make sure es5 browser support is required.

Reply:
Yeah true though this is solely to update tsconfig target to es2015 which is one of the criteria’s to enable differential loading.

At the point we don’t really need any knowledge of differential loading is enabled or not.

Anyways, this is obsolete, since we will not update the current projects to es2015.